dao/src/main/java/com/carhouse/dao/mappers/CarSaleMapper.java

```diff
@@ -34,6 +34,10 @@
      * The constant DATE.
      */
     private static final String DATE = "date";
+    /**
+     * The constant IMAGE.
```

Comment:
not sure it's required, only checkstyle maybe.
might be removed with checkstyle rule.